components/SingleSelectRadio/styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from 'styled-components';
+import COLORS from '@/styles/colors';
+
+interface SelectedProps {
+  isSelected: boolean;
+}
+
+export const ComponentContainer = styled.div<SelectedProps>`
+  display: flex;
+  flex-direction: row;
+  gap: 0.9375rem;
+  background-color: white;
+  border: 0.125rem solid transparent;
```

Comment:
There should be a light grey border for non-selected inputs. see designs 
<img width="380" alt="Screen Shot 2024-11-11 at 8 41 13 PM" src="https://github.com/user-attachments/assets/b5554b9e-bba2-43ab-b316-0f4f05be240b">